app/views/recipes/show.html.erb

```diff
@@ -3,6 +3,36 @@
 <%= render @recipe %>
 
 <div style="display: flex; justify-content: space-between">
-  <%= link_to "Generate Shopping List", new_recipe_path, class: 'btn btn-primary', style:"margin-bottom: 12px" %>
-  <%= link_to "Add Ingredient", new_recipe_path, class: 'btn btn-primary', style:"margin-bottom: 12px" %>
-</div>
\ No newline at end of file
+  <%= link_to "Generate Shopping List", "#{shopping_list_index_path}?recipe_id=#{@recipe.id}", data: { turbo_frame: "modal" }, class: 'btn btn-primary', style:"margin-bottom: 12px" %>
+  <%= link_to "Add Ingredient", new_recipe_food_path, class: 'btn btn-primary', style:"margin-bottom: 12px" %>
+</div>
+
+<table class="table table-striped">
+  <thead>
+  <tr>
+    <th>Food</th>
+    <th>Quantity</th>
+    <th>Value</th>
+    <th>Actions</th>
+  </tr>
+  </thead>
+  <tbody>
+  <% @recipe_food.each do |ingredient| %>
```

Comment:
:red_circle: I think this table should contain the foods for a specific recipe. As you can see this will list all the recipe_foods items. Kindly make sure you use the relationship between the recipe and recipe_food to create this table 
This line should be similar to this :arrow_down: 
` <% @recipe.recipe_foods.each do |ingredient| %>`